addons/calendar/models/calendar.py

```diff
@@ -1256,7 +1256,7 @@ def _rrule_serialize(self):
         """
         if self.interval and self.interval < 0:
             raise UserError(_('interval cannot be negative.'))
-        if self.count and self.count <= 0:
+        if self.count <= 0:
```

Comment:
Not sure if we need to test if self.count is not None before.
@nim-odoo what do you think ?

Reply:
What is the value of `self.count` in this case?

Reply:
0

Reply:
si it ends up being falsy and does not trigger the initial condition for UserError

Reply:
By the way, an addtionnal comment would be nice because the reason why this works is not clear.

Reply:
```
# if self.count and ... :
        # ends up being false count is 0
        if hasattr(self, 'count') and self.count <= 0:
```

Reply:
ok. `self.count <= 0` should be enough, then. It should always be an `int`

Reply:
side note: from https://github.com/odoo/odoo/blob/3e2ee43fd0aa6a06d5d5ecd1ff669f4a09698330/odoo/fields.py#L1141